tests/test_product_requirements_text_QCI97.py

```diff
@@ -0,0 +1,32 @@
+# QCI-97
+
+# In the "Select products to deploy screen, there is a text box to the right
+# of the products list.   It as you select various products will contain text
+# that explains what is required by each product selected.   Be sure the text
+# is correct for the products selected.
+
+def test_product_requirements_text(new_deployment_pg, expected_text):
+    assert new_deployment_pg.\
+            get_requirements('general') == \
+            expected_text['product_selection_pg_req_box']\
+                ['general_expected_text']
```

Comment:
I'm unsure about this, splitting the line behind almost every word. Does this look right to you?